tests/test_app.py

```diff
@@ -0,0 +1,63 @@
+# test_db.py
+
+from app import app
+import unittest
+import os
+os.environ['TESTING'] = 'True'
+
+
+class AppTestCase(unittest.TestCase):
+    def setUp(self):
+        self.client = app.test_client()
+
+    def test_home(self):
+        response = self.client.get('/')
+        assert response.status_code == 200
+        html = response.get_data(as_text=True)
+        assert "<title>About</title>" in html
```

Comment:
@marleneharms You need to add the TODO tasks you were asked: 
Add more tests relating to the home page